regexes/client/browsers.yml

```diff
@@ -1727,6 +1754,9 @@
   version: '$1'
   engine:
     default: 'WebKit'
+- regex: 'Chrome.+UC Browser(\d+[\.\d]+)'
+  name: 'UC Browser'
+  version: '$1'
```

Comment:
```yml
engine:
    versions:
      28: 'Blink'
```

Reply:
I can't add this, because the UC Browser version is less than 28, and it still adds WebKit. I've added Blink as default.